src/Rules/MethodExistsVisitor.php

```diff
@@ -0,0 +1,56 @@
+<?php declare(strict_types = 1);
+
+namespace WPCompat\PHPStan\Rules;
+
+use PhpParser\Node;
+use PhpParser\NodeVisitorAbstract;
+use PHPStan\Node\Printer\ExprPrinter;
+use function count;
+
+final class MethodExistsVisitor extends NodeVisitorAbstract {
```

Comment:
I tried several variants, even those mentioned in https://github.com/phpstan/phpstan/discussions/11648

I went with the visitor, as I got the impression that this `SinceVersionRule` is expected to work on the `$requiresAtLeast` parameter of `SinceVersionRule`, no matter which version of wordpress itself is installed at the time of analysis.

Reply:
Yep that's correct 👍